src/renderer/containers/main/editor.ts

```diff
@@ -193,6 +193,8 @@ class Editor extends Container<EditorState, MainCTX> {
 
     if ( this.isSplit () ) return;
 
+    if ( editing && this.isEditing () ) return;
+
```

Comment:
that makes no sense. `this.isEditing()` returns the value of `this.state.editing` and `editing` is set to the negation of that value. There is no way this branch will ever get hit.

Reply:
> and editing is set to the negation of that value.

only when the value is not set explicitly though.